src/heretic/r_data.c

```diff
@@ -735,7 +735,15 @@ int R_TextureNumForName(const char *name)
 
     i = R_CheckTextureNumForName(name);
     if (i == -1)
-        I_Error("R_TextureNumForName: %s not found", name);
+    {
+        // [crispy] fix absurd texture name in error message
```

Comment:
We don't need this to terminate the name after 8 chars. Just put `%.8s` in the format string. I just didn't know this at that time.